sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamSqlDslBase.java

```diff
@@ -133,4 +147,25 @@ public void preparePCollections(){
 
     return rows;
   }
+
+  private static List<BeamRecord> prepareInputRowsInTableB() throws ParseException{
```

Comment:
Would`TestUtils.RowsBuilder` be useful here? For example something like this:

```java
TestUtils.RowsBuilder
            .of(rowType)
            .addRows(
               3.0, 1.0, 1.0, 3, 1,
               4.0, 2.0, 2.0, 4, 2,
               5.0, 3.0, 1.0, 5, 3,
               6.0, 4.0, 2.0, 6, 4,
               8.0, 4.0, 1.0, 8, 4
            );

Reply:
move test to BeamSqlDslAggregationCovarianceTest